### ISSUE TITLE
[10]How to turn off the automatic reset password function

### ISSUE THREAD

**Issue by lonelyleaves** [OCA/server-tools#1083]
After installing password security, will be forced to request a reset password after login, how to close, thank you！

**Comment by pedrobaeza** [comment]
@lasley I'm a bit annoyed about this also. If I have a user with password, I don't want to reset my password next time I login

**Comment by lasley** [comment]
Is this for all users? Basically all existing passwords are considered expired?

**Comment by pedrobaeza** [comment]
Yes

**Comment by lonelyleaves** [comment]
Can you explain the function of this reset password I did not understand

**Comment by lasley** [comment]
The point of the password reset is to allow for the expiry of passwords based on criteria, which is intended functionality for this module and described in depth in the ReadMe. 

Fix sounds simple enough- will just need to add a post install hook to set the last reset date of all users to the current date. I’m assuming it’s just blank in the db, which is going to trigger the expiry. 


-Dave Lasley

**Comment by pedrobaeza** [comment]
I understand the principle, but mixed with the fact that you can't repeat a password makes everyone to change the password, even if the old one is secured enough. I prefer to not reset it.

**Comment by lasley** [comment]
@pedrobaeza - yeah I figured you got the principle, but @lonelyleaves was asking so I elaborated.

PR #1084

**Comment by lasley** [comment]
I think this actually also affected new users. Not positive though & wasn't necessarily worth checking.

**Comment by rven** [comment]
This will also fix the issue I've got.
When installing the module on a DB all users passwords are expired.
But when logging in, the login attempt fails and nothing happens, the pop-up is not shown but redirected to the login page again.
It is not possible to login to Odoo after installing this module in an existing environment.
I fixed it with the following update: `update res_users set password_write_date=current_date;`
Exactly the same what is done in this Bug Fix.
:+1:
